root/types.js

```diff
@@ -547,6 +547,19 @@ declare type IswcT = {|
   +work_id: number,
 |};
 
+declare type KnockoutObservable<T> = {
+  [[call]]: (() => T) & ((T) => empty),
+  peek: () => T,
+  subscribe: ((T) => void) => empty,
```

Comment:
This is to fix the issue with function statics being an empty object now instead of `any`. Knockout observables are functions, hence the `[[call]]` type, but they also expose "methods" like peek, subscribe, push, remove, etc.

Reply:
I used `empty` to indicate that the return value shouldn't be used in those cases.